modules/cluster-logging-deploy-multitenant.adoc

```diff
@@ -0,0 +1,43 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * assemblies/cluster-logging-deploying.adoc
+
+[id="cluster-logging-deploy-multitenant_{context}"]
+= Installing cluster logging into a multitenant network
+
+[role="_abstract"]
+If you are deploying cluster logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
+
+Because the Elasticsearch Operator and the Cluster Logging Operator are installed in different projects, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. How you allow this access depends on how you configured multitenant isolation mode.
+
+.Procedure
+
+1. To allow traffic between the Elasticsearch Operator and the Cluster Logging Operator, perform one of the following:
++
+* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *Multitenant* mode, use the following command to join the two projects:
```

Comment:
This bullet point, and accompanying lines 19-25, doesn't apply to OSD, and should be removed.

Reply:
Thanks, Will. Done.